aws/rust-runtime/aws-config/src/profile/parser/normalize.rs

```diff
@@ -161,6 +183,42 @@ pub(super) fn merge_in(
                 .sso_sessions
                 .entry(name.to_string())
                 .or_insert_with(|| SsoSession::new(name, Default::default())),
+            SectionKey::Other {
+                key: section_type,
+                name: section_name,
+            } => {
+                // services section
+                let sections = base
+                    .other_sections
+                    // get section by section type
+                    .entry(section_type.to_string())
+                    // if it doesn't exist, create new hashmap for that section type
+                    .or_default();
+                // get section by section name
+                let named_section = sections
+                    .entry(section_name.to_string())
+                    // if it doesn't exist, create new hashmap for that section name
+                    .or_default();
+
+                for (sub_properties_name, raw_sub_properties) in &raw_profile {
+                    tracing::info!("[{section_type} {section_name}].{sub_properties_name} = {raw_sub_properties}");
+                    // TODO do we need a more-specific validator for service IDs here?
```

Comment:
I think aws-config definitely isn't the place to do that validation since someone could use an old version of aws-config with a newly launched SDK crate, and it should work in that case.

The SDK crates also can't validate it though since they only know about their own individual service. So, really, I don't think any kind of validation can be done here.

Reply:
yeah we can only validate based on general profile file syntax